lighthouse-core/report/templates/report-template.html

```diff
@@ -245,8 +247,41 @@ <h2 class="config-section__title">Blocked URL Patterns</h2>
 {{/each}}
 
 <script>
-window.addEventListener('DOMContentLoaded', _ => {
-  new LighthouseReport(window.lhresults);
+(function(exports) {
+'use strict';
+
+/**
+ * Note: getFilenamePrefix is duplicated from asset-saver.js
```

Comment:
I'm up for suggestions on this too. We could factor out `getFilenamePrefix` from asset-saver.js and make it work in both Node and the browser.